README.md

```diff
@@ -75,6 +77,18 @@ Please refer to [changelog.md](docs/en/changelog.md) for details and release his
 
 Differences between MMSelfSup and OpenSelfSup codebases can be found in [compatibility.md](docs/en/compatibility.md).
 
+### Preview of 1.x version
+
+A brand new version of **MMSelfSup v1.0.0rc1** was released in 01/09/2022:
+
+Highlights of the new version:
```

Comment:
more highlights can be listed here, for example, 
- support MAE with different pre-training epochs and backbones of different scales
- more concise API
- higher accurcy for some algorithms